doc/en/components/grids/grid/overview.md

```diff
@@ -107,22 +133,22 @@ Now that we have the grid packages imported, let’s get started with the basic
 <igc-grid id="grid1" auto-generate="true"></igc-grid>
 ```
 
-```ts
+```typescript
 constructor() {
-    let grid1 = (document.getElementById("grid1") as IgcGridComponent);
+    let grid1 = document.getElementById("grid1") as IgcGridComponent;
     grid1.data = data;
 }
 ```
 
 The `Id` property is a string value and is the unique identifier of the grid which will be auto-generated if not provided, while **data** binds the grid, in this case to local data.
```

Comment:
These describe the snippet(s) above though so the idea is for them to match, right? To that end ideally this would be `id`, but as far as I can tell this is used for all platforms and is somehow picked up as API links for [Blazor](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/grids/grid/overview) - I'm assuming there's a way to mention an API member in a platform-agnostic way and was going to ask if we can use it here.

Also, for consistency `data` should also be marked as a code ref (`` `data/Data` ``) rather than being bolded.